### PR TITLE
Also publish sbt 1.0.1 version

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,11 +1,22 @@
+val commonSettings = Seq(
+  crossSbtVersions := Seq("0.13.16", "1.0.1"),
```
Comment (jonas) [inline build.sbt]: Maybe bump to 1.0.2

File: project/Dependencies.scala [hunk 2]
```diff
@@ -3,7 +3,8 @@ import sbt._
 object Version {
   val foundation = "6.3.1"
   val prettify   = "4-Mar-2013-1"
-  val sbtParadox = "0.2.12"
+  // Needs https://github.com/lightbend/paradox/pull/147
```
Comment (jonas) [inline project/Dependencies.scala]: I suggest to remove the comment now that it has been released.

File: build.sbt [hunk 3]
```diff
@@ -1,11 +1,22 @@
+val commonSettings = Seq(
+  crossSbtVersions := Seq("0.13.16", "1.0.2"),
```
Comment (ktoso) [inline build.sbt]: 1.0.3 here would be better

Comment (raboof) [reply build.sbt]: 1.0.3 isn't out yet right? that's 1.0.2 :)

Comment (ktoso) [reply build.sbt]: Ah those pesky one-off-errors! Thanks :)

Comment (jonas) [reply build.sbt]: Though you might just be living in the future. :-P

**Pull request merged**